jobs/build-arch.Jenkinsfile

```diff
@@ -70,6 +70,8 @@ def cosa_img = params.COREOS_ASSEMBLER_IMAGE ?: pipeutils.get_cosa_img(pipecfg,
 
 def stream_info = pipecfg.streams[params.STREAM]
 
+def cosa_controller_img = stream_info.cosa_controller_img ?: cosa_img
```

Comment:
this probably needs an appropriate comment about the context like the comment from the code block we're removing.

Reply:
I was thinking the comment block deleted here would be moved (and reworded) to RHCOS' `config.yaml` above `cosa_build_arch_job_controller_img_hack` since that's the only place we'd actually configure it. WDYT?

Reply:
I guess that's sufficient. In that case let's add a comment in the docs/config.yaml about RHCOS being the only consumer of this right now and we'll be able to remove it in the future.